queries.sql

```diff
@@ -6,3 +6,64 @@ select (name,escape_attempts) from animals where weight_kg > 10.5;
 select * from animals where neutered is true;
 select * from animals where name != 'Gabumon';
 select * from animals where weight_kg >= 10.4 and weight_kg <= 17.3;
+
+-- Day 2
+
+begin;
+update animals set species = 'unspecified';
+select * from animals;
+rollback;
```

Comment:
- [ ] After the rollback on line 15, you should add a query to verify that the change was undone.